parts/linux/cloud-init/nodecustomdata.yml

```diff
@@ -277,6 +277,49 @@ write_files:
     [Install]
     WantedBy=multi-user.target
 
+- path: /etc/default/kubelet
+  permissions: "0644"
```

Comment:
should probably harden these but I don't _think_ these files were part of the CIS benchmark

Reply:
touche, I don't think so but maybe a good follow up